tests/e2e/tools/lib/covers.js

```diff
@@ -106,7 +106,7 @@ async function checkCoversBlock(page, style) {
   // Check Load more button
   const loadMoreBtn = await page.$('button.load-more-btn');
   if (loadMoreBtn) {
-    await expect(page.locator('button.load-more-btn')).toHaveText('Read more');
+    await expect(page.locator('button.load-more-btn')).toHaveText('Load more');
```

Comment:
@mardelnet I think that we should change this text to make the specs working again, since this PR is blocking other PRs. We are only touching its spec, not the block functionality!

See what is going on https://output.circle-artifacts.com/output/job/608801f4-f131-49ac-9e4b-f2e9d05393f6/artifacts/0/usr/artifacts/playwright/report/index.html#?testId=62dc1fe6c3ba2c74379e-0a5f1c3f20abc1629ec0 .

In addition we can follow up specs whenever the Cover's migration is finished.

Reply:
I don't understand why we change this, the button text is changed [here](https://github.com/greenpeace/planet4-master-theme/blob/3d089f9746865effc3cb401dd679a4fabb83edfc/tests/e2e/tools/lib/covers.js#L69) and the point of the test is to check that this attribute `readMoreText` is working as expected if changed

Reply:
If the test fails without this change of text, it means the `readMoreText` behaviour is broken

Reply:
I suspect it takes the Covers's v2. Let me re-check it again.